src/inc/regdisp.h

```diff
@@ -8,7 +8,7 @@
 
 #ifdef DEBUG_REGDISPLAY
 class Thread;
-
+struct REGDISPLAY;
```

Comment:
arm64 build was successful. error is for something else. Checking this in to unblock nuget package publish. Will investigate later where x64 seems to be getting forward declaration from.